tests/unit/controllers-author.test.js

```diff
@@ -0,0 +1,97 @@
+const sinon = require('sinon');
+const authorController = require('../../src/controllers/author');
+const helpers = require('../../src/controllers/helpers');
+
+describe('author controller', () => {
+  const req = {
+    body: {},
+    params: {
+      authorId: 0
+    }
+  };
+  const res = {};
+
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  describe('create', () => {
+    it('is called correctly', async () => {
+      try {
+        const createMock = sinon.mock(helpers);
+
+        createMock.expects('create').once().withArgs(req.body, res, 'author');
+
+        await authorController.create(req, res);
+
+        createMock.verify();        
+      } catch(err) {
+        throw new Error(err);
+      }
+    });
+  });
+
+  describe('readAll', () => {
+    it('is called correctly', async () => {
+      try {
+        const readAllMock = sinon.mock(helpers);
+
+        readAllMock.expects('readAll').once().withArgs(res, 'author');
+
+        await authorController.readAll('', res);
+
+        readAllMock.verify();        
+      } catch(err) {
+        throw new Error(err);
+      }
+    });
+  });
+
+  describe('readById', () => {
+    it('is called correctly', async () => {
+      try {
+        const readByIdMock = sinon.mock(helpers);
+
+        readByIdMock.expects('readById').once().withArgs(req.params.authorId, res, 'author');
+
+        await authorController.readById(req, res);
+
+        readByIdMock.verify();        
+      } catch(err) {
+        throw new Error(err);
+      }
+    });
+  });
+
+  describe('update', () => {
+    it('is called correctly', async () => {
+      try {
+        const updateMock = sinon.mock(helpers);
+
+        updateMock.expects('update').once().withArgs(req.body, req.params.authorId, res, 'author');
```

Comment:
Theres a few instances where you have particularly long lines like this. Depending on your linter settings this could be written like so to make it easier to read at a glance:
```
        updateMock
          .expects("update")
          .once()
          .withArgs(req.body, req.params.authorId, res, "author");
```

This could be done very easily and automatically using prettier, see this guide [here](https://blog.bitsrc.io/add-prettier-to-your-project-d7e91ac03d05). How you setup the `.prettierrc` file will determine how you prroject will be formatted